### ISSUE TITLE
Suggestion: expose TipID as new keyword for notification templates

### ISSUE THREAD

**Issue by danilort** [globaleaks/GlobaLeaks#1514]
Hello,
I suggest a new keyword for the notification templates %TipID%.

Using %TipID% I can create a direct URL to to access the tip. Example:

notification template:
View TIP using Tor Browser:  %TipTorURL%
View TIP using https:  https://www.xxxx.yyy/#/status/%TipID%

the email:
View TIP using Tor Browser:  http://v3xxxxxxxxxxxxx3.onion/#/status/31804fcf-f665-4b02-a118-4c25430ed3a4
View TIP using https:  https://www.xxxx.yyy/#/status/31804fcf-f665-4b02-a118-4c25430ed3a4

Best regards

**Comment by evilaliv3** [comment]
Good suggestion @danilort this way user will be able to overcome eventual limits of the default templates and enrigh their own.

Already default template enforce the following:

```
The submission can be accessed:
via Tor at: %TorURL%
via Tor2web at: %T2WURL%
```

apart of the nice addition of exporting the TipID variable, why you were not founding sufficient to use the  %T2WURL% variable?

**Comment by danilort** [comment]
I do not use tor2web. The Receivers access with https (stunnel).

**Comment by evilaliv3** [comment]
got the point, thanks @danilort for the feedback.
given the easiness of such an addition it will be surely available by the end of the week

**Comment by evilaliv3** [comment]
implemented @danilort 

will be shipped by today with release 2.60.121
